rules/windows/defense_evasion_disable_windows_firewall_rules_with_netsh.toml

```diff
@@ -21,31 +21,33 @@ note = """## Triage and analysis
 The Windows Defender Firewall is a native component which provides host-based, two-way network traffic filtering for a
 device, and blocks unauthorized network traffic flowing into or out of the local device.
 
-Attackers can disable firewall rules which are intended to prevent lateral movement and command and control traffic to
-enable their operations.
+Attackers can disable the Windows firewall or its rules to enable lateral movement and command and control activity.
 
-This rule identifies patterns related to disabling firewall rules using the `netsh.exe` utility.
+This rule identifies patterns related to disabling the Windows firewall or its rules using the `netsh.exe` utility.
 
 #### Possible investigation steps
 
-- Identify the user account which performed the action and whether it should perform this kind of action.
+- Identify the user account that performed the action and whether it should perform this kind of action.
 - Contact the user to check if they are aware of the operation.
-- Investigate the script execution chain (parent process tree).
-- Investigate other alerts related to the user/host in the last 48 hours.
-- Analyze the executed command to determine what it allowed.
+- Investigate the process execution chain (parent process tree) for unknown processes. Examine their executable files
+for prevalence, whether they are located in expected locations, and if they are signed with valid digital signatures.
+- Investigate other alerts associated with the user/host during the past 48 hours.
 
 ### False positive analysis
 
-- This mechanism can be used legitimately. Check whether the user is legitimately performing this kind of activity.
-- Assess the need to disable the modification of the rule, and whether these actions expose the environment to
-unnecessary risks.
+- This mechanism can be used legitimately. Check whether the user is an Administrator and is legitimately performing
+troubleshooting.
+- In case of an allowed B-TP, assess adding rules allowing needed traffic and re-enable the firewall.
```

Comment:
```suggestion
- This mechanism can be used legitimately. Check whether the user is an administrator and is legitimately performing
troubleshooting.
- In case of an allowed benign true positive (B-TP), assess adding rules to allow needed traffic and re-enable the firewall.
```